doc/src/intrinsics-memory.md

```diff
@@ -0,0 +1,92 @@
+# Challenge 2: Verify the memory safery of core intrinsics using raw pointers
+
+- **Status:** *One of the following: [Open | Resolved | Expired]* Open
+- **Tracking Issue:** *Link to issue*
+- **Start date:** *YY/MM/DD*
+- **End date:** *YY/MM/DD*
+
+-------------------
+
+
+## Goal
+
+Annotate Rust core::intrinsics functions that manipulate raw pointers with their safety contract and verify their usage in the standard library are in fact safe.
+
```

Comment:
Looks like you decided not to include an extended Description or Motivation section?

Reply:
Yes, keeping it simple for now.